packaging/rpm/crc.spec.in

```diff
@@ -30,7 +30,7 @@ Name:           %{goname}
 Release:        1%{?dist}
 Summary:        CRC's main executable
 License:        APL 2.0
-ExcludeArch:    armv7hl i686
+ExcludeArch:    armv7hl i686 ppc64le
 URL:            %{gourl}
 ExcludeArch:    s390x
```

Comment:
Why is this separate?

Reply:
No idea, was there before but I have now removed one of that.